### ISSUE TITLE
Invalid `main` field in `package.json`

### ISSUE THREAD

**Issue by ozio** [thlorenz/find-parent-dir#7]
Since nodejs 16.0.0 has been released, this message started to appear:

```
(node:31065) [DEP0128] DeprecationWarning: Invalid 'main' field in '/Users/oz/n/lib/node_modules/npm-check/node_modules/find-parent-dir/package.json' of 'find-parent-dir.js'. Please either fix that or report it to the module author
```

**Comment by vBm** [comment]
There's a pull request (#5) since 2018 that fixed the issue but it got never merged since the project is probably abandoned.

**Comment by spencermountain** [comment]
@thlorenz anything we can do to help?
thanks

**Comment by thlorenz** [comment]
Fixed in #5
